meta-mender-tegra/recipes-mender/tegra-state-scripts/files/switch-rootfs

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -e
+echo "$(mender show-artifact): $(basename "$0") was called!" >&2
```

Comment:
@zach-welch-aquabyte Maybe a bit unrelated, but on my device/image the binary is only available as `mender-update` and not as `mender`. Do you have any idea, why that could be the case?

Reply:
@apbr starting in Mender Client 4.0, the binary is split into `mender-auth` and `mender-update`. So unless additional configuration is applied, this script should actually refer to `mender-update`.

Reply:
Ok, I proposed a change: https://github.com/zach-welch-aquabyte/meta-mender-community/pull/1 (Includes also a first step towards compatibility with JP 6)

Reply:
As this PR is now merged, I created a PR directly here: https://github.com/mendersoftware/meta-mender-community/pull/402